kuma/users/templates/account/email.html

```diff
@@ -8,7 +8,7 @@
 
     {% if user.emailaddress_set.all().exists() %}
 
-      <p>{{ _('{email} is your <em>primary</em> email address')|fe(email=user.email) }}</p>
+      <p>{{ _('%(email)s is your <em>primary</em> email address'|safe, email=user.email) }}</p>
```

Comment:
I thought the reason we didn't need `|safe` was because of this in puente:
https://github.com/mozilla/puente/blob/master/puente/ext.py#L10-L12

But even testing with straight jinja2 this is also seems true:
```
>>> from jinja2 import Environment
>>> from django.utils import translation
>>> env = Environment(autoescape=True, extensions=['jinja2.ext.i18n'])
>>> env.install_gettext_translations(translation, newstyle=True)
>>> env.from_string("{{ _('%(foo)s is <em>bar</em>'|safe, foo=foo) }}").render(foo='test')
u'test is <em>bar</em>'
>>> env.from_string("{{ _('%(foo)s is <em>bar</em>', foo=foo)|safe }}").render(foo='test')
u'test is <em>bar</em>'
>>> env.from_string("{{ _('%(foo)s is <em>bar</em>', foo=foo) }}").render(foo='test')
u'test is <em>bar</em>'
```